src/System.Management.Automation/utils/PowerShellETWTracer.cs

```diff
@@ -1085,7 +1085,7 @@ public bool TraceJob(Job job)
             {
                 return DebugChannel.TraceDebug(PowerShellTraceEvent.Job,
                                                PowerShellTraceOperationCode.Method, PowerShellTraceTask.None,
-                                               job.Id.ToString(CultureInfo.InvariantCulture), job.InstanceId.ToString(), "NULL job");
+                                               "", "", "NULL job");
```

Comment:
the only reason i mention it is because of the use of String.Empty and this is consistent with that use